test/sourcing/Sourcing nvm.sh with alias for . should succeed

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+die () { echo "$@" ; exit 1; }
+
+alias .='cat'
+
+\. ../../nvm.sh
```

Comment:
the `\.` here will make the alias not do anything; we don't need to test `.` vs `\.`, what we need to test is the *install script* (not sourcing).

Reply:
So I have to define the alias in all files under `test/install_script/`, so that `install.sh` would always fail, if `.` is not escaped?

Reply:
not necessarily in all files, a single test would be better.